build/constants.yaml

```diff
@@ -152,12 +147,11 @@
     python310-version: 3.10.8
     python311-version: 3.11.0
     runtime-versions:
-    - 2.7
-    - 3.6
-    - 3.8
-    - 3.9
-    - 3.10
-    - 3.11
+      3.7: bullseye
+      3.8: bullseye
```

Comment:
Double checking if these should be `buster`? Since 3.9 is `buster`.

Reply:
According to the [`generateDockerfiles.sh` script for Python](https://github.com/microsoft/Oryx/blob/main/images/runtime/python/generateDockerfiles.sh#L24-L25), all of the runtime versions are based on `bullseye` _except_ for 3.9, which is based on `buster`.